test/clocksi_SUITE.erl

```diff
@@ -130,7 +132,7 @@ clocksi_test1(Config) ->
                       {read, {Key1, Type}}]]),
     ?assertMatch({ok, _}, Result2),
     {ok, {_, ReadSet2, _}}=Result2,
-    ?assertMatch([0,0], ReadSet2),
+    ?assertMatch([0,1], ReadSet2),
```

Comment:
For some reason, that test was "wrong". 
It tests that you read your updates within the same transaction. 
Whithin a tx, it: reads an object, updates it, and reads it again, and expects the value to be, wrongly, the same.

The previous path (mentioned in the above discussion), that used execute_tx had a bug that update operations were not applied.

In this case I believe we need to fix the test too.